controllers/backup.go

```diff
@@ -212,11 +212,11 @@ func (r *BackupReconciler) createBackupForResource(
 
 	err := r.Get(ctx, veleroResIdentity, veleroResBackup)
 	if err != nil {
-		backupLogger.Info(
+		backupLogger.Info(fmt.Sprintf(
 			"velero.io.Backup [name=%s, namespace=%s] returned error, checking if the resource was not yet created",
 			veleroResIdentity.Name,
 			veleroResIdentity.Namespace,
-		)
+		))
 		// check if this is a  resource NotFound error, in which case create the resource
```

Comment:
ok, thanks!
Can we have this key/values checked in with another task so we can look at all other log usages
I changed it because as I was testing the fix I noticed some log messages in this format : velero.io.Backup [name=%s, namespace=%s] returned error, checking ...